test/sparse/sparse.jl

```diff
@@ -716,6 +716,13 @@ end
         @test S1290[end] == (S1290[1] + S1290[2,2])
         @test 6 == sum(diag(S1290))
         @test Array(S1290)[[3,1],1] == Array(S1290[[3,1],1])
+
+        # check that indexing with an abstract array returns matrix
+        # with same colptr and rowval eltypes as input
```

Comment:
Could be worthwhile adding a breadcrumb to this PR and/or the issue.

Reply:
Added breadcrumbs on all the tests